titus-common/src/main/java/com/netflix/titus/common/network/client/RxRestClientConfiguration.java

```diff
@@ -31,4 +35,8 @@
 
     @DefaultValue("200")
     long getRetryDelayMs();
+
+    default Set<HttpResponseStatus> getNoRetryStatuses() {
```

Comment:
Is it possible to configure this via Archaius? Maybe it should be `Set<String>`, which would be mapped to `HttpResponseStatus` internally?

Reply:
What is the purpose of this method? Wouldn't retry behavior be dependent on the specific API contract that is being called?

Reply:
The purpose is to tell the Retryable HTTP client which codes to not retry on. Otherwise it retries on all (e.g., it makes 3 GET attempts when the first response is a 404). When a specific connector creates its HTTP client it specifies whatever values make sense based on its contract, which is what is done if nftitus.

I'll update to a List<String> for Archaius, though the default method is the recommended way to define the collection value (no DefaultValue for collections).